lib/hydra/works/services/virus_checker_service.rb

```diff
@@ -30,8 +30,9 @@ def local_path_for_file(file)
         return file.path if file.respond_to?(:path)
         Tempfile.open('') do |t|
           t.binmode
-          t.write(file.content.read)
-          file.content.rewind
```

Comment:
@escowles FileBody does not have a rewind, so I removed it.  https://github.com/projecthydra/active_fedora/blob/a641a8f0d65e7c01f395eeb58c5e87c638395e2e/lib/active_fedora/file/streaming.rb

I believe that each time your call the each method it reopens the connection to fedora:

https://github.com/projecthydra/active_fedora/blob/a641a8f0d65e7c01f395eeb58c5e87c638395e2e/lib/active_fedora/file/streaming.rb#L27